src/toast/ToastProvider.tsx

```diff
@@ -130,8 +163,7 @@ export const ToastProvider: React.FC<ToastProviderProps> = ({
         key={placement}
         className={`${className} ${className}--${placement}`}
         style={{
-          position: "fixed",
-          ...PLACEMENTS[placement],
+          ...(placementStyles[placement] || {}),
```

Comment:
Now users can specify the placement styles

```ts
<ToastProvider placementStyles={{ "bottom-left": { left: 0 } }} />
```

Reply:
Yes they could also use classNames (toast__container--left) to style too. but previously because it was hardcoded in the style we had to use !important on the classNames.